guide/getting-started/technology-primer.md

```diff
@@ -47,26 +47,9 @@ Illustration sources
 
 # Bitcoin technology primer
 
-It can take many years to understand Bitcoin in every detail, but luckily you don't need to know everything to design great products. Let's look at the most important ideas and technologies behind Bitcoin by answering some basic questions.
+It can take many years to understand bitcoin in every detail, but luckily you don't need to know everything to design great products. Let's look at the most important ideas and technologies behind bitcoin by answering some basic questions.
 
-## Is it Bitcoin or bitcoin?
```

Comment:
I'd keep this part because it is important for readers to understand that there's both bitcoin, the network, and bitcoin, the currency. Some slight re-wording should do the trick.

Reply:
Taking that one back. This is covered in the following paragraphs.